components/sysutils/lsof/Makefile

```diff
@@ -30,6 +31,9 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/justmake.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+# Need GNU version of yes
+PATH = $(PATH.gnu)
+
```

Comment:
That comment is a warning for other developers.  The illumos version of yes, at least the one supplied with oi_151a8, ignores SIGPIPE.  This causes yes to never terminate.  Here's part of my truss:

write(1, "\n\n\n\n\n\n\n\n\n\n\n\n".., 5120)    Err#32 EPIPE
    Received signal #13, SIGPIPE [ignored]
write(1, "\n\n\n\n\n\n\n\n\n\n\n\n".., 5120)    Err#32 EPIPE
    Received signal #13, SIGPIPE [ignored]
write(1, "\n\n\n\n\n\n\n\n\n\n\n\n".., 5120)    Err#32 EPIPE
    Received signal #13, SIGPIPE [ignored]

This only happens when `gmake build' is run from the components directory.  It works correctly when only that component is built.

Perhaps yes has been fixed in later versions of illumos.  Perhaps hipster is already running the GNU version of yes.

My solution was the change PATH so that the GNU version is always selected.